internal/metrics/smbstatus.go

```diff
@@ -31,10 +31,12 @@ type SmbStatusSigning struct {
 	Degree string `json:"degree"`
 }
 
-// SmbStatusShare represents a single entry from the output of 'smbstatus -S'
-type SmbStatusShare struct {
+// SmbStatusTCon represents a 'tcon' output field
```

Comment:
TCon is probably ok but T stands for Tree and I think 'SMBStatusTreeCon` would read nicer (also SMB is an initialism and the Go standard practice is to uppercase all initialisms)

Reply:
Fair enough. I will split it into two commits (within this PR):
1) Rename `SmbStatusTCon` --> `SmbStatusTreeCon`
2) Align all names with Go standard: `SmbStatus*` --> `SMBStatus*`